tests/test_utils_misc/test_return_with_argument_inside_generator.py

```diff
@@ -2,6 +2,7 @@
 import warnings
 from functools import partial
 from unittest import mock
+import ast
```

Comment:
It shouldn't be used in tests directly.

Reply:
Hiya! Good call out -- I amended my commit to use magic mock. Let me know what you think. Cheers.